cafebabel/static/css/styles.css

```diff
@@ -809,37 +811,13 @@ button.menu + nav a {
 }
 
 button.follow:before {
-    font-family: 'icomoon';
-    content: '\e9da';
-    font-size: 3rem;
+    content: 'Follow';
```

Comment:
Be careful that this text won't be translatable as it is in CSS

Reply:
hum..true :-)